### DISCUSSION TITLE
0.0.65

### DISCUSSION THREAD

**Discussion by Xzandro** [Xzandro/sw-exporter#531] [General]
* Fixed issues with building the Mac version, thanks @bradbeam for testing

<hr /><em>This discussion was created from the release <a href='https://github.com/Xzandro/sw-exporter/releases/tag/v0.0.65'>0.0.65</a>.</em>

**Comment by magichustle74** [comment]
I have been stumbling from on place to place due to not one of you who have the knowledge of how to get the rune optimizer to work being either willing are an at e not able to make one cohesive walkthrough that works I've been trying for months and even spent money on the peatron thinking if I spend money shurly the answers and after 11 plus years the lack of anyone's desire to help out the other man in this game and com2us rng getting ever more shitty by the day I will thank you for helping me to decide to quite the only game I've been playing over a decade fuck this game in out
